### PR TITLE
- modified readme

### LINKED ISSUE #32: Add 'How to contribute' section to readme

### PR DESCRIPTION
**Pull request by JanEberlein**
closes #32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,19 @@ One sentence description.
...
+1:	check for open code-reviews and do them if available
+2: 	choose an open issues
+3:	assign yourself
+4: 	create a local branch with name: issue-number (without hash) + "-" + issue-title (e.g. 8-setup-mono-repo)
+5: 	codecodecode
+6: 	commit + push
+7: 	in pull-requests: create pull-request for current changes (from your branch to master) (there should be a button)
+8: 	wait for at least 2 other contributers to review your code and for the automated pipeline to finish
+9: 	merge your pull-request
+
```
Comment (anbraten) [inline README.md:32]:
```suggestion
## How to contribute 

### For externals

TBD 🤔 

### For official team members

Always check for open [code-reviews](https://github.com/project-leek/project-leek/pulls?q=is%3Aopen+is%3Apr+type%3Apr+review%3Arequired) and do them if available before you start with a new issue.

#### How to start with a new issue

1. choose an open issues from the [project boards Todo column](https://github.com/project-leek/project-leek/projects/1)
1. 👉🏾 assign the issue to yourself 
1. create a local branch with name: issue-number (without hash) + "-" + issue-title (e.g. 8-setup-mono-repo)
1. 👨🏾 👩🏾 ✍🏾 💻 
1. ⬆️ commit + push 
1. in [pull-requests](https://github.com/project-leek/project-leek/pulls): create pull-request for current changes (from your branch to master) (there should be a button)
1. 🕙 wait for at least 2 other contributors to review your code and for the automated pipeline to finish
1. 🥗 merge your pull-request
1. 😃 be happy to be a part of this project
```

**Review by anbraten — CHANGES_REQUESTED** [review]

**Review by anbraten — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -17,6 +17,27 @@ One sentence description.
...
+Always check for open [code-reviews](https://github.com/project-leek/project-leek/pulls?q=is%3Aopen+is%3Apr+type%3Apr+review%3Arequired) and do them if available before you start with a new issue.
+
+#### How to start with a new issue
+
+1. choose an open issues from the [project boards Todo column](https://github.com/project-leek/project-leek/projects/1)
+1. 👉🏾 assign the issue to yourself 
+1. create a local branch with name: issue-number (without hash) + "-" + issue-title (e.g. 8-setup-mono-repo)
+1. 👨🏾 👩🏾 ✍🏾 💻 
+1. ⬆️ commit + push 
+1. in [pull-requests](https://github.com/project-leek/project-leek/pulls): create pull-request for current changes (from your branch to master) (there should be a button)
```
Comment (anbraten) [inline README.md:37]:
```suggestion
1. create a new pull-request
  1. go to [pull-requests](https://github.com/project-leek/project-leek/pulls)
  1. there should be a button to create a pull-request for your changes (from your branch to master)
  1. add a description to you pull-request starting with `closes #<number-of-your-issue>` exp: `closes #1234`  
```

**Review by anbraten — CHANGES_REQUESTED** [review]
sorry think it would be nice to add this change as well 😂

File: README.md [hunk 3]
```diff
@@ -17,6 +17,27 @@ One sentence description.
...
+
+### For official team members
+
+Always check for open [code-reviews](https://github.com/project-leek/project-leek/pulls?q=is%3Aopen+is%3Apr+type%3Apr+review%3Arequired) and do them if available before you start with a new issue.
+
+#### How to start with a new issue
+
+1. choose an open issues from the [project boards Todo column](https://github.com/project-leek/project-leek/projects/1)
+1. 👉🏾 assign the issue to yourself 
+1. create a local branch with name: issue-number (without hash) + "-" + issue-title (e.g. 8-setup-mono-repo)
```
Comment (schnickalodeon) [inline README.md:34]: That's not needed anymore
```suggestion
1. At the moment you assign the issue, a local branch with an auto-generated title will be created for you.
```

File: README.md [hunk 4]
```diff
@@ -17,6 +17,27 @@ One sentence description.
...
+1. 👨🏾 👩🏾 ✍🏾 💻
```
Comment (schnickalodeon) [inline README.md:35]: Thats not clear enough.
```suggestion
1. 👨🏾 👩🏾 ✍🏾 💻 ☕  (start coding)
```

**Review by schnickalodeon — CHANGES_REQUESTED** [review]
A few request changes...

**Review by schnickalodeon — APPROVED** [review]
That's ok

**Review by anbraten — APPROVED** [review]

**Pull request merged**